src/esp/gfx/CubeMap.cpp

```diff
@@ -201,64 +265,84 @@ void CubeMap::recreateTexture() {
   }
 }
 
-void CubeMap::recreateFramebuffer() {
-  Mn::Vector2i viewportSize{imageSize_, imageSize_};
-  for (int iFbo = 0; iFbo < 6; ++iFbo) {
-    frameBuffer_[iFbo] = Mn::GL::Framebuffer{{{}, viewportSize}};
-  }
+void CubeMap::recreateFramebuffer(unsigned int cubeSideIndex,
+                                  unsigned int framebufferSize) {
+  Mn::Vector2i viewportSize{int(framebufferSize),
+                            int(framebufferSize)};  // at mip level 0
```

Comment:
Code golf -- the single-argument constructor is there exactly for this:

```suggestion
  Mn::Vector2i viewportSize{int(framebufferSize)};  // at mip level 0
```

About the casts, I ended up using `Vector2i` instead of `Vector2ui` for sizes because the extra annoyance with unsigned types wasn't worth it. So maybe here just `int framebufferSize` could be fine as well.